src/core/api/models/webhook_policy.go

```diff
@@ -0,0 +1,32 @@
+package models
```

Comment:
We still cannot use one unique model for webhook?

Reply:
> We still cannot use one unique model for webhook?

yep, webhook model in api is different with model in dao.

Reply:
Why not, the model is for both dao and api otherwise why adding the `json` annotations...

Reply:
model unified